content/04.study.md

```diff
@@ -473,3 +473,20 @@ However, because this model was not subsequently tested on ground-truth datasets
 
 Method development for interpreting new types of sequencing data has historically taken two steps: first, easily implemented hard cutoffs that prioritize specificity over sensitivity, then expert development of probabilistic models with hand-developed inputs [@tag:Torracinta2016_sim].
 We anticipate that these steps will be replaced by deep learning, which will infer features simply by its ability to optimize a complex model against data.
+
+
+
+### Neuroscience
+
+Artificial neural networks were originally conceived as a model for computation in the brain.
+The deep artificial neural networks of today have evolved to become a workhorse in computer science, with broad applications across many fields outside of neuroscience.
+However, there is still a deep connection between deep networks and the study of the brain.
+The rich parallel history of artificial neural networks in computer science and neuroscience is reviewed in [@doi:10.3389/fncom.2016.00094; @doi:10.1101/133504].
+
+Convolutional neural networks were originally conceived as faithful models of visual information processing in the primate visual system, and are still considered so [@doi:10.1038/nn.4244].
+The activations of hidden units in consecutive layers of deep convolutional networks have been found to parallel the activity of neurons in consecutive brain regions involved in processing visual scenes.
+Such models of neural computation are called "encoding" models, as they predict how the nervous system might encode sensory information in the world.
+
+As with other fields of science, deep networks have also been useful as computational tools in neuroscience.
```

Comment:
Instead `As with other fields of science`, can we transition with a more direct contrast with the above paragraph?  Something like

> Even when they are not directly modeling biological neurons, deep networks have been a useful computational tool in neuroscience.